themes/oh-my-posh/oh-my-posh.theme.bash

```diff
@@ -0,0 +1,20 @@
+# shellcheck shell=bash
+
+if _command_exists oh-my-posh; then
+	export POSH_THEME=${POSH_THEME:-https://raw.githubusercontent.com/JanDeDobbeleer/oh-my-posh/v$(oh-my-posh --version)/themes/jandedobbeleer.omp.json}
+	eval "$(oh-my-posh --init --shell bash --config "${POSH_THEME}")"
+else
+	_log_warning "The oh-my-posh binary was not found on your \$PATH."
```

Comment:
Maybe we should make the warnings into a single warning? With many newlines?

I think it will look nicer

Reply:
well, I didn't have BASH_IT_LOG_LEVEL set at all, so it echos nothing to the screen, I didn't know how it looks. I'll test it now.

Reply:
you can use `bash-it doctor` to easily see the output!